testapps/test_pillowtop/tests/test_xform_pillow.py

```diff
@@ -0,0 +1,47 @@
+from unittest import skip
+from django.test import TestCase, override_settings
+from corehq.apps.es import FormES
+from corehq.form_processor.interfaces.processor import FormProcessorInterface
+from corehq.form_processor.utils import TestFormMetadata
+from corehq.form_processor.tests.utils import FormProcessorTestUtils
+from corehq.pillows.xform import XFormPillow
+from corehq.util.elastic import delete_es_index
+from corehq.util.test_utils import get_form_ready_to_save
+
+
+class XFormPillowTest(TestCase):
+
+    domain = 'xform-pillowtest-domain'
+
+    def setUp(self):
+        FormProcessorTestUtils.delete_all_xforms()
+        self.pillow = XFormPillow()
+        delete_es_index(self.pillow.es_index)
```

Comment:
yeah, that's probably good to add. a lot of our tests (including this one) clean up at the beginning rather than the end, but definitely doesn't hurt to do both.